src/js/components/navigation.js

```diff
@@ -79,6 +88,10 @@ const resize = () => {
 
 const onMenuClose = () => navigation.toggleNav.call(navigation, false);
 const hideActiveNavDropdown = () => {
+  if (!navActive) {
```

Comment:
This check used to be in line 91, 96, and 107. Moved it to `hideActiveNavDropdown()` instead, so it checks when the function is called.